materials/tasks.py

```diff
@@ -0,0 +1,38 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from datetime import timedelta
+from django.utils import timezone
+
+from config.settings import EMAIL_HOST_USER
+from materials.models import Subscription
+from users.models import CustomUser
+
+
+@shared_task
+def mail_update_course_info(course_id):
+    """Отправка сообщения об обновлении курса"""
+    subscription_course = Subscription.objects.filter(course=course_id)
```

Comment:
Так как нужны и курс и пользователь, более оптимально запрос будет выглядеть так:
```suggestion
    subscription_course = Subscription.objects.select_related('course', 'user').filter(course=course_id)
```